doc/source/whatsnew/v0.20.0.txt

```diff
@@ -79,3 +79,5 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
+
+- Numpy ``astype()`` caused `inf` values to be falsely converted to integers. Affected ``astype()`` in Series and DataFrames (:issue:`14265`)
```

Comment:
say something more like:

Bug in ``.astype()`` where ``inf`` values incorrectly converted to integers.